### ISSUE TITLE
Add objects support

### ISSUE THREAD

**Issue by ViieeS** [mattiash/angular-tablesort#108]
Needs to add objects support in `ts-repeat`

### Example

```js
$scope.data = {
  "123": { id: 123, name: "dave", age: 23 },
  "456": { id: 456, name: "chris", age: 23 },
  "789": { id: 789, name: "bob", age: 23 },
  "101": { id: 101, name: "tom", age: 23 },
  "102": { id: 102, name: "tim", age: 23 }
}
```

**Comment by ViieeS** [comment]
@mattiash It's pretty simple. I don't know why it wasn't done before :relieved:

**Comment by mattiash** [comment]
I don't think it is a good idea to treat an object as a list and iterate over its values. I think the application should transform it into a list before trying to display it in a table.

**Comment by ViieeS** [comment]
@mattiash AngularJS natively supports object iteration via ng-repeat [doc](https://docs.angularjs.org/api/ng/directive/ngRepeat#iterating-over-object-properties). Also I have no problem with this over 2 years of using my fork.
